bitnami/zookeeper/templates/secrets.yaml

```diff
@@ -33,8 +33,8 @@ metadata:
   {{- end }}
 type: Opaque
 data:
-  keystore-password: {{ default (randAlphaNum 10) .Values.tls.client.keystorePassword | b64enc | quote }}
-  truststore-password: {{ default (randAlphaNum 10) .Values.tls.client.truststorePassword | b64enc | quote }}
+{{ .Values.tls.client.keystorePasswordSecretKey | indent 2 -}}: {{ default (randAlphaNum 10) .Values.tls.client.keystorePassword | b64enc | quote }}
+{{ .Values.tls.client.truststorePasswordSecretKey | indent 2 -}}: {{ default (randAlphaNum 10) .Values.tls.client.truststorePassword | b64enc | quote }}
```

Comment:
I would restrict the `truststorePasswordSecretKey`/`existingSecretTruststoreKey` exclusively to the name of the key to be used in case an existingSecret is provided. Else, the default values are `keystore-password` and `truststore-password` (in other words, leave this file unchanged).

Reply:
Ok I fixed it and I added some helper functions to determine if we need to use `truststorePasswordSecretKey`/`existingSecretTruststoreKey` or `keystore-password`/`truststore-password`.